apparmor/fr.emersion.Mako

```diff
@@ -0,0 +1,22 @@
+#include <tunables/global>
+
+profile fr.emersion.Mako /usr/bin/mako {
+  #include <abstractions/base>
+  #include <abstractions/fonts>
+  #include <abstractions/wayland>
+
+  #include <abstractions/dbus-strict>
+  dbus bind
+       bus=session
+       name=org.freedesktop.Notifications,
+  dbus receive
+       bus=session
+       path=/fr/emersion/Mako
+       interface=fr.emersion.Mako,
+
+  /{run,dev}/shm/grim-* rw,
```

Comment:
This should be `mako-*` now on master.

Reply:
Thanks for the information... I am using the release. Fixed now.